test/test.js

```diff
@@ -76,16 +77,23 @@ test('stripFinalNewline in sync mode on failure', t => {
 	t.is(stderr, 'foo');
 });
 
+const getPathWithoutLocalDir = () => {
+	const newPath = process.env[PATH_KEY].split(path.delimiter).filter(pathDir => !BIN_DIR_REGEXP.test(pathDir)).join(path.delimiter);
+	return {[PATH_KEY]: newPath};
+};
+
+const BIN_DIR_REGEXP = /node_modules[\\/]\.bin/;
+
 test('preferLocal: true', async t => {
-	await t.notThrowsAsync(execa('ava', ['--version'], {preferLocal: true, env: {Path: '', PATH: ''}}));
+	await t.notThrowsAsync(execa('ava', ['--version'], {preferLocal: true, env: getPathWithoutLocalDir()}));
 });
 
 test('preferLocal: false', async t => {
-	await t.throwsAsync(execa('ava', ['--version'], {preferLocal: false, env: {Path: '', PATH: ''}}), {message: ENOENT_REGEXP});
```

Comment:
I am not completely sure why, but this fixes those tests on Windows.
Filtering the `.bin` directories from the `PATH` variable seems more adequate than making it completely empty as well.